data/deprecated.json

```diff
@@ -2031,5 +2031,13 @@
   {
     "old": {"industrial": "brickworks"},
     "replace": {"industrial": "brickyard"}
+  },
+  {
+     "old": {"cycleway": "opposite"},
+     "replace": {"oneway:bicycle": "no", "cycleway:both": "no"}
+  },
+  {
+     "old": {"cycleway:left": "opposite"},
```

Comment:
maybe for consistency there could also be a rule for [`cycleway:right=opposite`](https://taginfo.openstreetmap.org/tags/cycleway%3Aright=opposite#overview); I'm thinking of countries with left hand default traffic where this tag would have made sense?

Reply:
To me the ~500 times that is used does not merit adding it here. I think those need to be cleaned up with a MapRoulette Challenge or something similar.

But if you prefer, we can add it, of course. Ideally we should try to update the guidelines with something along the lines "low usage is OK when is "a system" of deprecations being added" (wording TODO).

Reply:
Ah, true. It also seems like much of the [data](https://overpass-turbo.eu/s/1Ynp) is not even in left hand traffic countries. So, a deprecation rule might be even less useful for those. Let's keep it as is. :+1: